style.css

```diff
@@ -282,3 +281,55 @@ h1 span {
   border-bottom: 8px;
   background-color: #ebf0ee;
 }
+
+.contact {
+  background: no-repeat 100% 100% url(./images/illustration-contact-form.png);
+  padding: 64px 24px 64px 24px;
+}
```

Comment:
- [ ] I also suggest to you to use 100% width for big screens

Reply:
Hi @elmar8287 when you say big screen do you mean desktop version?

Reply:
Dear @iLynette like another device with a bigger display, as a tablet, notebook...

Reply:
Okay thank you @elmar8287 I'm working on it